src/patternfly/components/Accordion/accordion.scss

```
@@ -152,6 +161,18 @@
     transition-property: opacity;
   }
 
+  &:is(:hover, :active),
```

I'd probably change this to `:focus-within`? Though it's not that big of a deal, we'd typically add this border to the interactive element as a best practice (https://core-staging.patternfly.org/guidelines#hover-styles). So it would be on the `__toggle` for hover/focus, then on the `__item` for expanded. wdyt?

```suggestion
  &:is(:hover, :focus-within),
```